### PR TITLE
added plugin front-end resources

### PR DESCRIPTION
**Pull request by timwright12**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "presets": [
+        "env"
```
Comment (daveross) [inline .babelrc]: @timwright12 `env`? Where's your sense of adventure? 😇

**Comment by daveross** [comment]
I'm good with this. My only hesitation is it's a lot of things we may not need on plugins, like editor styles. But my preference for parity between the theme & plugin scaffolds is winning out. And since we're bundling, it's not like the extra files are adding overhead if they aren't used.

File: assets/css/frontend/style.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/**
+ * TenUpScaffold
+ * https://project-website.tld
+ *
+ * Copyright (c) 2017 10up
```
Comment (daveross) [inline assets/css/frontend/style.css]: @timwright12 We should take out the copyright notice

File: includes/functions/core.php [hunk 3]
```diff
@@ -72,3 +72,42 @@ function activate() {
...
+ *
+ * @return void
+ */
+function scripts() {
+
+	wp_enqueue_script(
+		'frontend',
+		TENUP_SCAFFOLD_TEMPLATE_URL . "/dist/js/frontend.min.js",
+		[],
+		false,
```
Comment (daveross) [inline includes/functions/core.php]: @timwright12 We should set some kind of version number for the enqueues, even if we just use `TENUP_SCAFFOLD_VERSION`. It'll help with cache busting when we deploy upgrades.

**Comment by daveross** [comment]
Ok, I lied. Left a couple notes for you. Nothing major.

**Comment by timwright12** [comment]
@daveross Removed the copyright stuff and added a version to the CSS/JS

**Pull request merged**